sdk/monitor/azquery/custom_client.go

```diff
@@ -160,8 +160,9 @@ type LogsQueryOptions struct {
 	Wait *int
 }
 
-// String implements the fmt.Stringer interface for type LogsQueryOptions.
-func (l LogsQueryOptions) String() string {
+// preferHeader converts LogsQueryOptions from struct to properly formatting sting
```

Comment:
```suggestion
// preferHeader converts LogsQueryOptions from struct to properly formatted string
```

Reply:
fixed in [e48b6c9](https://github.com/Azure/azure-sdk-for-go/pull/19946/commits/e48b6c941d9c1e27bc3f3c0aea767646b1694985)